travis.yml.template

```diff
@@ -97,8 +97,12 @@ jobs:
       before_script:
         - |
           if ! git diff --name-only "$TRAVIS_COMMIT_RANGE" -- | grep -Eq "Dockerfile$"; then
-            echo "Skip deployment as none of the Dockerfiles has been changed."
-            exit
+            if git diff --name-only "$TRAVIS_COMMIT_RANGE" -- | grep -Eq "^generate-stackbrew-library.sh$"; then
```

Comment:
Is there a reason why to use nested `if` instead of an `&&`?

Reply:
Won't `&&` make it too long and then we'll need to wrap it?

Reply:
Yes but that would look very similar without the extra `if`

Reply:
actually the `if` looks more readable to me, prefer not to make the statement longer and then wrap it.

Reply:
Alright, doesn't really matter to me!